src/vis/scatter/utils.ts

```diff
@@ -26,6 +26,18 @@ export function isScatter(s: IVisConfig): s is IScatterConfig {
   return s.type === ESupportedPlotlyVis.SCATTER;
 }
 
+function calculateDomain(domain: [number | undefined, number | undefined], vals: number[]): [number, number] {
+  if (domain[0] && domain[1]) {
```

Comment:
@dvzacharycutler You will have to be more specific here, as 0 is a falsy value. I.e. you will need to explicitly check for `domain[0] == null`. 

Generally, I would rewrite this as follows: 

```js
return [domain[0] == null ? Math.min(...(vals as number[])) : domain[0], domain[1] == null ? Math.max(...(vals as number[])) : domain[1]]
```
If you need the `max + max / 20`, just extract the two as variables and return as array accordingly.